fs/unionfs/fs_unionfs.c

```diff
@@ -2441,6 +2520,50 @@ static int unionfs_stat(FAR struct inode *mountpt, FAR const char *relpath,
   return ret;
 }
 
+/****************************************************************************
+ * Name: unionfs_chstat
+ ****************************************************************************/
+
+static int unionfs_chstat(FAR struct inode *mountpt, FAR const char *relpath,
+                          FAR const struct stat *buf, int flags)
+{
+  FAR struct unionfs_inode_s *ui;
+  FAR struct unionfs_mountpt_s *um;
+  int ret;
+
+  finfo("relpath: %s\n", relpath);
```

Comment:
This debug log should be moved after the `DEBUGASSERT` for `relpath != NULL`.

Reply:
It's safe here since the pointer isn't dereferenced.

Reply:
Ok, I've checked that our internal vsprintf implementation checks for a NULL pointer and prints `(null)`.